src/styles/site.css

```diff
@@ -34,7 +34,6 @@ body {
   font-family: 'Soehne', sans-serif;
   -webkit-font-smoothing: antialiased;
   font-size: 1.4rem;
-  cursor: url('../../assets/images/cursors/black.svg'), auto;
```

Comment:
@Oluwa-nifemi why did you take this out?

Reply:
Because the it was affecting the cursor for the cover pages and that file contains general site styling but the special cursor is only used on the index and about page so it doesn't really qualify as general site styling